engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/util/GraphQLUtil.kt

```diff
@@ -49,9 +49,15 @@ fun newGraphQLError(
 fun toGraphQLError(
     raw: JsonMap,
 ): GraphQLError {
-    return newError()
+    val errorBuilder = newError()
         .message(raw["message"] as String?)
-        .build()
+    raw["extensions"]?.let { extensions ->
+        errorBuilder.extensions(extensions as JsonMap)
+    }
+    raw["path"]?.let { path ->
```

Comment:
Should we forward `location` too?

On that topic, this is a generic `JsonMap` -> `GraphQLError` so we _could_ change it to something like

```
fun toGraphQLError(
  map: JsonMap,
  retainPath: Boolean = true,
  retainLocation: Boolean = true,
): GraphQLError {
  if (retainPath) raw["path"]?.let { errorBuilder.path(path as AnyList) }
  return errorBuilder.build()
}
```

etc. and let the caller decide what to keep.

**But** we don't have to make this change for now, let's just merge it as is and fix it up later.